packages/graphql-elasticsearch-transformer/src/resources.ts

```diff
@@ -382,7 +382,8 @@ export class ResourceFactory {
     /**
      * Create the Elasticsearch search resolver.
      */
-    public makeSearchResolver(type: string, nonKeywordFields: Expression[], nameOverride?: string, queryTypeName: string = 'Query') {
+    public makeSearchResolver(type: string, nonKeywordFields: Expression[],
+        primaryKey: string, nameOverride?: string, queryTypeName: string = 'Query') {
```

Comment:
I think we can remove the ```queryTypeName``` parameter, I only found a single caller to this method and for the ```Query``` type name we've a utility function to get it.

Reply:
changed in the latest commit